src/components/Dashboard/ProjectActivity/PaymentActivity.tsx

```diff
@@ -60,98 +59,88 @@ export function PaymentActivity({
     return override ? override : e.note
   }
 
-  useEffect(() => {
-    setLoading(true)
```

Comment:
looks like this `setLoading(true)` was removed from the ProjectActivity components, which still needs to be there. this is a callback to set a `loading` state on the parent component

as an aside, this pattern has always felt a bit awkward to me, so if you have any ideas for streamlining it with this new `useSubgraphQuery` pattern then feel free to play around!